src/opm/output/eclipse/AggregateUDQData.cpp

```diff
@@ -249,18 +359,24 @@ const std::vector<int> Opm::RestartIO::Helpers::igphData::ig_phase(const Opm::Sc
                                                                    const std::vector<int>& inteHead
                                                                   )
 {
+    std::vector<const Opm::Group2*> curGroups(ngmaxz(inteHead), nullptr);
+    for (const auto& group_name : sched.groupNames(simStep)) {
+        const auto& group = sched.getGroup2(group_name, simStep);
+        int ind = (group.name() == "FIELD")
+            ? ngmaxz(inteHead)-1 : group.insert_index()-1;
```

Comment:
The `Group::insert_index()` has offset zero, so I do not think you should have `-1` here?!